dashboard/app/views/layouts/_unsupported_browser.html.haml

```diff
@@ -1,13 +1,43 @@
 %script{src: asset_path('browser-detector.js')}
 
-#warning-banner{style: 'display: none; height: 25px;'}
-  %i.fa.fa-warning.warning-sign
+#warning-banner{style: 'display: none'}
+
+  %i.fa.fa-warning.warning-sign.bannerIcon#warningIcon{style: 'display: none'}
+  %i.fa.fa-close.close-button.bannerIcon#dismissIcon{style: 'display: none; cursor: pointer', onclick: '$("#warning-banner").slideUp();'}
   &nbsp;
-  != t("compatibility.upgrade.unsupported_message", supported_browsers_url: 'https://support.code.org/hc/en-us/articles/202591743')
+  #unsupportedBrowser.bannerText{style: 'display: none'}!= t("compatibility.upgrade.unsupported_message", supported_browsers_url: 'https://support.code.org/hc/en-us/articles/202591743')
+  #appLabUnsupportedTablet.bannerText{style: 'display: none'}!= t("compatibility.upgrade.applab_unsupported_tablet")
+  #gameLabUnsupportedTablet.bannerText{style: 'display: none'}!= t("compatibility.upgrade.gamelab_unsupported_tablet")
+  #webLabUnsupportedBrowser.bannerText{style: 'display: none'}!= t("compatibility.upgrade.weblab_unsupported_browser")
```

Comment:
I know I bring this up all the time but I twitch a little when a file uses both. Do we want IDs and classes to be always lowerCamelCase or kebab-case? Let's just decide. lol